.gitignore

```diff
@@ -33,3 +33,6 @@ xcuserdata
 /documentation
 test/fixtures/api/assets.zip
 test/fixtures/storage/assets.zip
+/.idea
+/cmake-build-debug
```

Comment:
If it's something that you only see locally due to the use of your toolchain, you can also add these folders in a system-wide .gitignore file:

`~/.gitconfig`
```
[core]
	excludesfile = /Users/USERNAME/.gitignore
```